src/Filter/ScssPHP.php

```diff
@@ -55,6 +55,9 @@ public function input($filename, $input)
         }
         $sc = new Compiler();
         $sc->addImportPath(dirname($filename));
+        foreach ($this->_settings['paths'] as $path) {
```

Comment:
Shouldn't this option have a default value in `_settings`. Right now it looks like this would cause an error if scss wasn't configured with this option.